coreneuron/nrniv/netcvode.cpp

```diff
@@ -471,7 +473,7 @@ void NetCon::deliver(double tt, NetCvode* ns, NrnThread* nt) {
     nt->_t = tt;
 
     // printf("NetCon::deliver t=%g tt=%g %s\n", t, tt, pnt_name(target_));
-    POINT_RECEIVE(typ, target_, u.weight_index_, 0);
+    (*crnrn.get_pnt_receive()[typ])(target_, u.weight_index_, 0);
```

Comment:
better to pass typ to `crnrn.get_pnt_receive()`?

Reply:
yes, like many, many other instances all over the code. Let's defer this to when the data is in the correct class and accessed the way it actually should.